egs/gale_arabic/s5b/local/gale_data_prep_txt.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright 2014 QCRI (author: Ahmed Ali)
+# Apache 2.0
+
+galeData=$(readlink -f "${@: -1}" );  # last argumnet; the local folder
+txt_dvds=${@:1:${#}-1} # all the txt cds correspoding to the audio corpus; check text=( in ../run.sh
+
+
+top_pwd=`pwd`
+txtdir=$galeData/txt 
+mkdir -p $txtdir; cd $txtdir
+
+for cdx in $txt_dvds; do
+  echo "Preparing $cdx"
+  if [[ $cdx  == *.tgz ]] ; then
+     tar -xvf $cdx
+  elif [  -d "$cdx" ]; then
+    ln -s $cdx `basename $cdx`
+  else
+    echo "I don't really know what I shall do with $cdx " >&2 
+  fi
+done
+
+find -L . -type f -name *.tdf | while read file; do 
+sed '1,3d' $file 
```

Comment:
please add a comment saying what this sed command does, most people only use it for the s command.